data/settings/src/main/java/com/google/jetpackcamera/settings/model/CameraAppSettings.kt

```diff
@@ -26,8 +26,7 @@ data class CameraAppSettings(
     val flashMode: FlashMode = FlashMode.OFF,
     val captureMode: CaptureMode = CaptureMode.MULTI_STREAM,
     val aspectRatio: AspectRatio = AspectRatio.NINE_SIXTEEN,
-    val previewStabilization: Stabilization = Stabilization.UNDEFINED,
-    val videoCaptureStabilization: Stabilization = Stabilization.UNDEFINED,
+    val stabilizationMode: StabilizationMode = StabilizationMode.AUTO,
```

Comment:
That's a good point. I suppose we just want to push preview stabilization as the default since it has the lowest amount of side effects (though high-quality seems to still be WYSIWYG on the two premium devices I tested, and I didn't notice any latency).

I've removed the fallback to high-quality.